anaconda.spec.in

```diff
@@ -195,6 +195,40 @@ The anaconda-install-env-deps metapackage lists all installation environment dep
 This makes it possible for packages (such as Initial Setup) to depend on the main Anaconda package without
 pulling in all the install time dependencies as well.
 
+%package install-img-deps
+Summary: Installation image specific dependencies
+# This package must have no weak dependencies.
+Requires: udisks2-iscsi
+Requires: libblockdev-plugins-all >= %{libblockdevver}
+# active directory/freeipa join support
+Requires: realmd
+Requires: isomd5sum >= %{isomd5sumver}
+%ifarch %{ix86} x86_64
+Requires: fcoe-utils >= %{fcoeutilsver}
+%endif
+# likely HFS+ resize support
+%ifarch %{ix86} x86_64
+%if ! 0%{?rhel}
+Requires: hfsplus-tools
+%endif
+%endif
+# kexec support
+Requires: kexec-tools
+Requires: createrepo_c
+# run's on TTY1 in install env
+Requires: tmux
+# install time crash handling
+Requires: gdb
+Requires: rsync
+# only WeakRequires elsewhere and not guaranteed to be present
+Requires: device-mapper-multipath
+Requires: kpartx
```

Comment:
Are we actually using this ? Grepping through the Anaconda code for "kpartx" does not turn up anything & it only seems to ship a binary called "kpartx", so we should be calling it somewhere if we use it:

```
$ rpm -ql kpartx
/usr/lib/.build-id
/usr/lib/.build-id/4e
/usr/lib/.build-id/4e/07b79f248f194b8085ca74f9d32b3832abfc5c
/usr/lib/udev/kpartx_id
/usr/lib/udev/rules.d/11-dm-parts.rules
/usr/lib/udev/rules.d/66-kpartx.rules
/usr/lib/udev/rules.d/68-del-part-nodes.rules
/usr/sbin/kpartx
/usr/share/doc/kpartx
/usr/share/doc/kpartx/README
/usr/share/licenses/kpartx
/usr/share/licenses/kpartx/GPL-2.0
/usr/share/man/man8/kpartx.8.gz
```
Maybe we need the udev rules ? In any case I think it makes sense to drop a commend above it when we find out.

Reply:
Brian @bcl  mentioned that one in [the bug](https://bugzilla.redhat.com/show_bug.cgi?id=1893019). I am not sure if adding that one is correct, actually, and hoped someone could tell...

Reply:
I haven't noticed that you add new dependencies. Could you create a new commit for that or mention it in the commit message, please? Based on the bug, `device-mapper-multipath` seems to be fine, but I am not so sure about `kpartx`.

Reply:
Let's leave it out for now. I'll ask Brian.